pkg/controller/role/role_controller.go

```diff
@@ -123,8 +124,8 @@ func (r *ReconcileRole) Reconcile(request reconcile.Request) (reconcile.Result,
 		roleArn := *createOutput.Role.Arn
```

Comment:
Same note about checking createOutput.Role for nil before accessing its properties.